contracts/AccountIngress.sol

```diff
@@ -1,4 +1,18 @@
-pragma solidity 0.5.9;
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+pragma solidity >=0.7.0 <0.9.0;
```

Comment:
The code was written for solidity 0.5.9 originally and has seen almost no changes since then. In particular, no changes that would require a newer compiler. So while bumping the upper version bounds to `<0.9.0` is great, I would prefer to keep the lower bound at `>= 0.5.9`.

Allowing a wider version range makes it easier to combine the permission contract with other solidity code that carries a different version range without having to change the code.

Reply:
I think some of the changes to make it compile with 0.8 mean it doesn't compile any more with 0.5 or 0.6?

Reply:
https://docs.soliditylang.org/en/latest/060-breaking-changes.html

I think best case we can specify `pragma solidity >=0.6.0 <0.9.0;`

Reply:
Upgraded to @antonydenyer suggestion. 
It is not possible to maintain the 0.5.9 due to override modifier required since 0.6.0 and not accepted by 0.5.9.